source/linux/Industrial_Protocols/PTP/_PTP.rst

```diff
@@ -1847,7 +1847,7 @@ Procedure
 ~~~~~~~~~
 
 The script file
-`setup\_hsr.sh <http://processors.wiki.ti.com/images/1/18/Setup_hsr.pdf>`_
+`setup\_hsr.sh <http://processors.wiki.ti.com/images/1/18/Setup_hsr.pdf>`__
```

Comment:
@StaticRocket, while unrelated to this PR, this link does not work anymore. @danish-ti, could you please identify the equivalent for this link and create a PR to fix the link?

Reply:
@Siddharth-Vadapalli-at-TI  I don't think there is any equivalent for this link. To fix this script will need to be hosted somewhere and link to that should be added here. But as @cshilwant mentioned that this file is not used by any device, I think dropping this is the best option.

Reply:
@danish-ti I was only referring to `source/system/Demo_User_Guides/Servo_Drive_Demo_User_Guide.rst` file to @StaticRocket -  https://github.com/TexasInstruments/processor-sdk-doc/pull/125#discussion_r1886847478

Not sure about `source/linux/Industrial_Protocols/PTP/_PTP.rst`